<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgPlay</class>
 <widget class="QMainWindow" name="DlgPlay">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>652</width>
    <height>415</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>拼图          ——by：orall</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnGameStart">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>200</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文琥珀</family>
     </font>
    </property>
    <property name="text">
     <string>开始游戏</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnGameRules">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>240</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文琥珀</family>
     </font>
    </property>
    <property name="text">
     <string>游戏说明</string>
    </property>
   </widget>
   <widget class="QLabel" name="labTitle">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>361</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文新魏</family>
      <pointsize>19</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>拼图1.0   ——by：orall</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnImportPic">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>280</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文琥珀</family>
     </font>
    </property>
    <property name="text">
     <string>导入图片</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>50</y>
      <width>121</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>方正舒体</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-width: 3px;
border-style: solid;
border-color: rgb(0, 0, 0)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-width: 2px;
border-style: solid;
border-color: rgb(0,0,0)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>80</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-width: 2px;
border-style: solid;
border-color: rgb(0, 0, 0)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>80</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-width: 2px;
border-style: solid;
border-color: rgb(0, 0, 0)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>170</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-width: 2px;
border-style: solid;
border-color: rgb(0, 0, 0)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>170</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-width:2px;
border-style: solid;
border-color: rgb(0, 0, 0)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>170</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-width: 2px;
border-style: solid;
border-color: rgb(0, 0, 0)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>260</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-width: 2px;
border-style: solid;
border-color: rgb(0, 0, 0)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>260</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-width: 2px;
border-style: solid;
border-color: rgb(0, 0, 0)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>260</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-width: 2px;
border-style: solid;
border-color: rgb(0, 0, 0)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnExit">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>320</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文琥珀</family>
     </font>
    </property>
    <property name="text">
     <string>退出游戏</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>652</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
